rootfs/common/INSTALL_PREFIX/scripts/logpull/collect/qca.sh

```diff
@@ -31,12 +31,12 @@
 
 collect_qcawl()
 {
-    collect_cmd iwconfig
+    collect_cmd iw dev
```

Comment:
There's already some level of support in device-core for this. I would much prefer device-core to be extended instead of duplicated in platform-qca. Can you take a look at that please?